proto/EntryProtocol.proto

```diff
@@ -0,0 +1,230 @@
+syntax = "proto3";
```

Comment:
Could you add Alibaba Apache-2.0 license header?

Reply:
added in https://github.com/pingcap/ticdc/pull/649/commits/01a5a3ef9510784039fa58e35da42e0b142c81a9